mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Rights.test

```diff
@@ -38,3 +38,12 @@ S: a6 NO DELETEACL failed. Mailbox not found.
 
 C: a7 SETACL INBOX bob lr
 S: a7 OK SETACL completed.
+
+C: a8 SETACL INBOX anyone p
+S: a8 OK SETACL completed.
```

Comment:
You are not getting the ACL after each SETACL in order to asser the changed done?